src/Utilities/tests/for_testing/test_checkVector.cpp

```diff
@@ -10,6 +10,7 @@
 //////////////////////////////////////////////////////////////////////////////////////
 
 #include "catch.hpp"
+#include "OhmmsPETE/TinyVector.h"
```

Comment:
I don't feel it is needed here. Can it be removed?

Reply:
Done

Reply:
This one can't be removed.  Either it has to be here or in checkVector.hpp and I remember a review comment removing it from that file.